### PR TITLE
@lennamau :tested for repair item

### PR DESCRIPTION
**Pull request by arunabenji29**

### CODE REVIEW

**Comment by arunabenji29** [comment]
@lennamau :MVP

**Comment by lennamau** [comment]
## Rating (1-3)
### 3

File: enhancing/enhancer.js [hunk 1]
```diff
@@ -6,17 +6,49 @@ module.exports = {
 };
 
 function succeed(item) {
-  return { ...item };
+  if(item.enhancement<20){
+    return { ...item,enhancement:item.enhancement+1 };
+  }
+  return {...item}
 }
```
Comment (lennamau) [inline enhancing/enhancer.js:14]: ✔

File: enhancing/enhancer.spec.js [hunk 2]
```diff
@@ -1,2 +1,50 @@
...
+    describe('repair()', () => {
+        it('check if the durability is 100', () => {
+            // console.log('inside the repair test',enhancer.repair({durability:89}).durability)
+            expect(enhancer.repair({durability:89}).durability).toBe(100);
+            expect(enhancer.repair({durability:-89}).durability).toBe(100);
+            expect(enhancer.repair({durability:100}).durability).toBe(100);
+            expect(enhancer.repair({durability:0}).durability).toBe(100);
+        });
+    });
+
```
Comment (lennamau) [inline enhancing/enhancer.spec.js:14]: ✔

File: enhancing/enhancer.spec.js [hunk 3]
```diff
@@ -1,2 +1,50 @@
...
+    describe('get()', () => {
+        it('chnages the name, if the enhancement value is above zero', () => {
+            expect(enhancer.get({enhancement:0,name:'thorns'}).name).toBe('thorns');
+            expect(enhancer.get({enhancement:1,name:'Knife'}).name).toBe('[+1] Knife');
+            expect(enhancer.get({enhancement:19,name:'Soup'}).name).toBe('[+19] Soup');
+        });
+    });
+
+
```
Comment (lennamau) [inline enhancing/enhancer.spec.js:49]: ✔

**Pull request merged**